app/views/docs/badge_view.rb

```diff
@@ -78,7 +78,7 @@ def view_template
 
   def components
     [
-      Docs::ComponentStruct.new(name: "PhlexUI::Badge", source: "https://github.com/PhlexUI/phlex_ui/blob/main/lib/phlex_ui/badge.rb", built_using: :phlex)
```

Comment:
If we are using `Badge` maybe we should specify that `include Phlex` need to be in `application_view.rb` or use `Phlex::Badge`

Reply:
@iseth , sure! I will try to find the remaining "PhlexUI::" references and add to the installation guide the instruction to `include Phlex` into `application_view.rb`

Anything else?

Reply:
<img width="785" alt="Screenshot 2024-07-06 at 9 54 31 AM" src="https://github.com/PhlexUI/PhlexUI-web/assets/727781/45dfdfe6-368c-456a-8391-d1e327f6d54b">

@iseth , I added the instruction to add PhlexUI module.